packages/providence/providence.conf.js

```diff
@@ -0,0 +1,98 @@
+// This file is read by dashboard and cli and needs to be present under process.ced()
```

Comment:
process.cwd()?

Reply:
I saw it as well. Will be fixed in next push